cli/__init__.py

```diff
@@ -18,7 +22,50 @@ class ContentType(str, Enum):
     IMAGES = "images"
 
 
+class Environment(str, Enum):
+    PRODUCTION = "production"
+    STAGING = "staging"
+    DEVELOPMENT = "development"
```

Comment:
I am not convinced that `development` is an environment - if it is, I don't think it's a natural sibling of `production` or `staging`.

It's more like a different workflow, rather than an environment (and reading the PR, this is true for its implementation too): I might antagonise that it should be a separate option altogether?

If that doesn't sound appealing, I think it could be renamed to something that tells you where stuff is happening, like `rank-cluster`.